4.6.2/Dockerfile

```diff
@@ -1,5 +1,14 @@
-FROM microsoft/iis
+FROM microsoft/dotnet-framework:4.6.2
 
-RUN powershell -Command Add-WindowsFeature NET-Framework-45-ASPNET; \
-    powershell -Command Add-WindowsFeature Web-Asp-Net45; \
-    powershell -Command Remove-Item -Recurse C:\inetpub\wwwroot\*
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
```

Comment:
Given that this has already been released, reverting it is likely to cause more headaches for people who have updated their dockerfiles to counter this change, so I'd leave it as-is